recipes/roary/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
+  number: 2
```

Comment:
And you need to add r-base and probably a few r-libraries to the run dependencies

Reply:
Ok, I'll check. Thanks!